packages/assets/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@atb-as/generate-assets",
+  "version": "0.0.1",
+  "private": false,
+  "description": "OOS Design System generate-assets",
+  "license": "EUPL-1.2",
+  "bin": "./lib/bin.js",
+  "main": "./lib/generate.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "prepublish": "tsc"
```

Tenker meg om tror jeg kanskje denne burde være `prepare`
```suggestion
    "prepare": "tsc"
```

---

Kan være lurt å legge til en `build` script her også. F.eks heller ha det slik:

```
  "scripts": {
    "prepare": "yarn build",
    "build": "tsc"
  },
```

For lerna er satt opp til å kunne bygge alle pakker og sjekke TSC ut i fra script som er definert under build. Lerna er systemet som håndterer monorepo-organiseringen

---

Tenker meg om så holder det faktisk med `build` og ikke `prepare` i det hele tatt. Lerna er satt opp til å kjøre build-all på alle pakker når det køres `lerna pub` for å publisere til npm. Da blir pakken her bygget.

---

Da endte jeg opp med denne, så burde det slå til når `lerna publish` eller `lerna run build` kjører.

```
  "scripts": {
    "build": "tsc"
  },
```